src/viem/estimateFeesPerGas.ts

```diff
@@ -9,7 +9,7 @@ import {
 export async function estimateFeesPerGas(
   client: Client,
   feeCurrency?: Address
-): Promise<{ maxFeePerGas: bigint; maxPriorityFeePerGas: bigint; baseFeePerGas: bigint }> {
+): Promise<{ maxFeePerGas: bigint; maxPriorityFeePerGas: bigint; baseFeePerGas: bigint | null }> {
```

Comment:
`baseFeePerGas` from `getBlock` now has a type of `BigInt | null`, it used to be `any` in prior versions. Looks like this will always be returned for the latest block, it's only null for older blocks before the EIP 1559 upgrade https://www.quicknode.com/docs/ethereum/eth_getBlockByHash. Another option to fix this would be to just throw if this is `null` as it would never happen in practice for the latest block.

Reply:
I have a slight preference for throwing here, since we're only dealing with EIP-1559 based TXs in the rest of the code.